AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/databean/ActionType.java

```diff
@@ -4,23 +4,45 @@
 import java.io.Serializable;
 
 public enum ActionType implements Serializable {
-  COMMON("Common"),
-  FINANCE("Finance"),
-  FOOD_AND_DRINK("Food And drink"),
-  HEALTH_AND_FITNESS ("Health and fitness"),
-  TRANSPORTATION("Transportation");
+  COMMON("Common", "common"),
+  FINANCE("Finance", "finance"),
+  FOOD_AND_DRINK("Food And drink", "food-and-drink"),
+  HEALTH_AND_FITNESS("Health and fitness", "health-and-fitness"),
+  TRANSPORTATION("Transportation", "transportation");
 
   private String name;
+  private String url;
 
-  ActionType(String name) {
+  ActionType(String name, String url) {
     this.name = name;
+    this.url = url;
   }
 
   public static ActionType getActionTypeValue(int idx) {
     return ActionType.values()[idx % 5];
   }
 
+  public static ActionType getActionTypeByName(String intentName) {
+    if (intentName.equals("actions.intent.OPEN_APP_FEATURE") || intentName.equals("actions.intent.GET_ACCOUNT") || intentName.equals("actions.intent.GET_ORDER") || intentName.equals("actions.intent.GET_THING")) {
```

Comment:
maybe we can put action names of each category in a set and use set.contains() to check, it makes the code more readable

Reply:
Fixed.